ExampleGallery/UAP/CameraEffectExample.xaml.cs

```diff
@@ -27,12 +28,15 @@ public sealed partial class CameraEffectExample : UserControl, ICustomThumbnailS
         const string noEffect = "No effect";
         const string displacementEffect = "Displacement effect";
         const string rotatingTilesEffect = "Rotating tiles effect";
+        const string variableEffect = "Variable effect";
```

Comment:
suggest naming this "Emboss effect" rather than "Variable effect" - follow the existing pattern of describing the visual effect it creates in a way that will make sense to a user running the sample